hledger-web/Foundation.hs

```diff
@@ -297,19 +299,11 @@ getLastMessage = cached getMessage
 addform :: Text -> ViewData -> HtmlUrl AppRoute
 addform _ vd@VD{..} = [hamlet|
 
-<script language="javascript">
```

Comment:
I removed this attribute because it is deprecated: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script